### ISSUE TITLE
Create the route to handle the UPDATE to any of the site's page contents

### ISSUE THREAD

**Issue by molinacarlosdev** [thaiscmky/phonow#71]

**Comment by thaiscmky** [comment]
@molinacarlosdev Content pages CMS bit deferred to later release. @RealTayy  will be hardcoding any content that isn't part of the restaurant menu. This can be revisited once the rest of the MVP is done
